vsdenoise/dfttest.py

```diff
@@ -129,8 +129,6 @@ def __init__(
     ) -> None:
         if isinstance(locations, Mapping):
             frequencies, sigmas = list(locations.keys()), list(locations.values())
-        elif len(locations) % 2:
```

Comment:
I don't know why this was here. Passing in any range of tuples seems perfectly reasonable to me.

This only thing that needs to be even is the actual tuples themselves, not the list of said tuples.

I ran into this while testing this change.